javascript/modules/client/test/integration/client-addressList/decoding.test.ts

```diff
@@ -0,0 +1,127 @@
+// @ts-ignore
+declare const describe, it, expect;
+
+import {
+  ClientAddressList,
+  Context,
+  ContextPlugin,
+  IPluginSettings,
+} from "../../../src";
+import { contextParamsLocalChain } from "../constants";
+
+describe("Action decoders", () => {
```

Comment:
Shouldn't we add a higher level `describe("Client Address List")` to encapsulate these tests together?
Same everywhere